Dockerfile

```diff
@@ -3,21 +3,24 @@ FROM busybox:1.31.1-glibc
 ARG BUILD_DATE
 ARG VCS_REF
 
+ENV WAIT_TIME_SECONDS 4
```

Comment:
Wondering if this is required actually... (to be defined in the Dockerfile)

Reply:
This ENV can't be overriden in this manner as far as I recall.

You could either:

1 - Set hte env from an arg (seems overkill)
2 - Just default it inside the docker-entrypoint script

```
WAIT_TIME_SECONDS=${WAIT_TIME_SECONDS:-"4"}

Reply:
I think ignore me...